samples/net/lwm2m_client/prj.conf

```diff
@@ -39,3 +39,8 @@ CONFIG_NET_CONFIG_MY_IPV4_GW="192.0.2.2"
 # Longer endpoint name might be returned in a registration reply
 CONFIG_COAP_EXTENDED_OPTIONS_LEN=y
 CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE=40
+# This has to be match length of LWM2M_APP_ID and if LWM2M_APP_ID is empty,
+# then this has to be match length of CONFIG_BOARD. Default 16 is not enough
+# for some boards, so, increase it to 32.
+CONFIG_LWM2M_SECURITY_KEY_SIZE=16
```

Comment:
Shouldn't this be set to 32?  I'd also rather see this in `overlay-dlts.conf`

Reply:
oops, was trying to excite assert and forgot to revert, what about https://github.com/zephyrproject-rtos/zephyr/pull/60272#issuecomment-1631493471?